ReadMe.md

```diff
@@ -0,0 +1,25 @@
+# OP2Helper
+
+The OP2Helper library is used to ease development of Outpost 2 missions. It does this by wrapping the Outpost2DLL library, and builds functionality on top of the exported methods. While the Outpost2DLL library is aimed at providing the raw exports from Outpost2.exe (and only the raw exports), the OP2Helper library contains custom code to build functionality around the exported methods.
```

Comment:
Very minor, I think it should read "and building functionality" for the tense to be consistent.

Reply:
I assume you mean the first instance. I'm having trouble seeing this one.

Reply:
I think it should read:

It does this by wrapping the Outpost2DLL library, and **building** functionality on top of the exported methods

That would keep the tense the same across the sentence.

I would just merge if you don't agree or don't see it. Not worth spending any more time on.

Reply:
You're probably right. That section does seem like it could be reworded and simplified. And yeah, the tense does seem a bit off.